tokio-stream/src/wrappers/broadcast.rs

```diff
@@ -10,6 +10,28 @@ use std::task::{ready, Context, Poll};
 
 /// A wrapper around [`tokio::sync::broadcast::Receiver`] that implements [`Stream`].
 ///
+/// # Example
+///
+/// ```
+/// use tokio::sync::broadcast;
+/// use tokio_stream::wrappers::BroadcastStream;
+/// use tokio_stream::StreamExt;
+///
+/// #[tokio::main]
+/// async fn main() -> Result<(), Box<dyn std::error::Error>> {
```

Comment:
I like to avoid this error type in examples because it is not `Send` which means it only works in the `block_on` task.

Reply:
Avoiding unwrap is fine, I just prefer a different error type. E.g., it could be `Box<dyn Error+Send+Sync>`.

Reply:
I addressed this by changing the result type to the actual result, and generally hiding the main function / return values in all examples.